### DISCUSSION TITLE
[Lists] Star needed?

### DISCUSSION THREAD

**Discussion by qdequippe** [community/community#7764] [Mobile] [answered]
Hi,

First this new Lists feature is really cool :)

Is is possible to either add quickly a repository to a list and star it automatically or add a repo to a list without star it?

Regards

**Comment by esahmed21** [comment, marked as answer]
Hey, Thank you for reaching out! Currently it's not possible to add not starred repository to a list. You need first to star the repository and then add to list button will appear. Quickly adding a repository to a list and starring it automatically is a great addition, I'll make sure to pass your feedback to the team.

**Comment by EricBatlle** [comment]
The "add to list without star it" for the own repos could be a nice feature to have! :D
